DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/classes/DSACampaign.java

```diff
@@ -14,38 +14,69 @@
 
 package com.google.sps.classes;
 
-// contains all the necessary information about a DSA campaign, including id's, input settings, and statistics
+// contains all the necessary information about a DSA campaign
 public class DSACampaign {
 
-	public int DSACampaignId;
-    public int userId;
-    public int keywordCampaignId;
+    // unique identifier for the DSA campaign
+	public String DSACampaignId;
```

Comment:
Why int instead of string?

Reply:
I'm using string instead of int.

Reply:
Sorry I meant why string instead of int for id?

Reply:
The user id's are sometimes too long to hold in ints, and we were having issues with longs.

Reply:
Oh interesting, what exactly are the issues with longs?

Reply:
I'm not sure, Neel mentioned that he was having some issue with doubles/longs. Either way, I think strings make more sense because we won't be performing any operations on the id's (constants).

Reply:
@neelbagora - What were the issues with doubles/longs?

And even so, we generally would want to use longs since comparison and space usage is more expensive on strings.

Reply:
Oh I think the user id's come as strings anyways as the default, so we would have to convert the string into a long instead of just keeping it as a string.